README.md

```diff
@@ -13,6 +13,9 @@ https://github.com/britzl/defold-sharing/archive/master.zip
 
 Or point to the ZIP file of a [specific release](https://github.com/britzl/defold-sharing/releases).
 
+<details><summary>You don't need to merge manifests manually starting Defold 1.2.153</summary>
```

Comment:
What's with the html tags?

Reply:
You can check it here https://github.com/agulev/defold-sharing